arch/xtensa/Kconfig

```diff
@@ -238,26 +238,14 @@ config XTENSA_CP_INITSET
 		is provided by CONFIG_XTENSA_CP_INITSET.  Each bit corresponds to one
 		coprocessor with the same bit layout as for the CPENABLE register.
 
-config XTENSA_DUMPBT_ON_ASSERT
-	bool "Dump backtrace on assertions"
-	default y
-	depends on DEBUG_ALERT
-	---help---
-		Enable a backtrace dump on assertions.
-
-config XTENSA_BTDEPTH
-	int "Backtrace depth"
-	default 50
-	depends on XTENSA_DUMPBT_ON_ASSERT
-	---help---
-		This is the depth of the backtrace.
-
 config XTENSA_INTBACKTRACE
-	bool "Full backtrace from interrupts"
-	default n
-	depends on XTENSA_DUMPBT_ON_ASSERT
+	bool
```

Comment:
how to disable this option if without the prompt string?

Reply:
If `CONFIG_SCHED_BACKTRACE` is enabled, this option is mandatory. I didn't remove it to enhance readability and make sure this feature exists.

Reply:
if so why not remove XTENSA_INTBACKTRACE and check SCHED_BACKTRACE directly?

Reply:
This feature (being able to correctly dump the backtrace during an interrupt) is not implemented for all archs. I didn't remove it to ease readability and make it easier to figure out that this feature exists for xtensas.

Reply:
Ok, could we support this feature on riscv too?

Reply:
I suppose so. It's on my to-do list...